src/leveled_log.erl

```diff
@@ -494,8 +501,11 @@ log_test() ->
     log_timer(d0001, [], os:timestamp()).
 
 log_warning_test() ->
-    ok = log(g0001, [], [warning, error]),
-    ok = log_timer(g0001, [], os:timestamp(), [warning, error]).
+    ok = log(g0001, [], [warning, error], ?LOGBASE, backend),
+    ok =
+        log_timer(
+            g0001, [], os:timestamp(), [warning, error], ?LOGBASE, backend
+        ).
```

Comment:
Add a test for the unknown key `wrongkey`:
`??? = log_timer(wrongkey, [], os:timestamp(), [warning, error], ?LOGBASE, backend)`
I think it should return ok and have no side effect action, similar to when the key would be present, but has a log level one is not interested in.

Reply:
See above.  I've removed the default, and so wrongkey will crash.  I've added a test to assertException to make this explicit.